sphinx_needs/directives/needimport.py

```diff
@@ -81,7 +82,7 @@ def run(self) -> Sequence[nodes.Node]:
         else:
             logger.info(f"Importing needs from {need_import_path}")
 
-            if not os.path.isabs(need_import_path):
+            if not posixpath.isabs(need_import_path):
```

Comment:
If you use posixpath here, then you need to use posix paths for the entire logic in the function,
and confirm this is the actual behaviour you want on Windows.
It would also be better to use `pathlib.PurePosixPath` (see https://docs.python.org/3/library/pathlib.html)

Reply:
Look at the `:download:` role and going for `self.env.relfn2path()` instead of the custom logic.
This also removes the deprecated behavior.